Cargo.toml

```diff
@@ -175,3 +176,12 @@ paseo-runtime-constants = { git = "https://github.com/polkadot-fellows/runtimes"
 # asset-hub-paseo-runtime = { git = "https://github.com/paseo-network/runtimes/", tag = "v1.2.5-system-chains", default-features = false }
 # paseo-runtime = { git = "https://github.com/paseo-network/runtimes/", tag = "v1.2.5-system-chains", default-features = false }
 # paseo-runtime-constants = { git = "https://github.com/paseo-network/runtimes/", tag = "v1.2.5-system-chains", default-features = false }
+
+ismp = { git = "https://github.com/r0gue-io/ismp", branch = "polkadot-v1.14.0", default-features = false }
```

Comment:
Worth noting the maintenance burden, as we now need to update multiple repos with SDK version uplifts.